components/docs-chef-io/content/automate/ha_chef_backend_to_automate_ha.md

```diff
@@ -17,276 +15,334 @@ gh_repo = "automate"
-```cmd
-     scp -i /path/to/key backup\_$(date '+%Y%m%d%H%M%s') user@host:/home/user
-```
+    - Execute the below command to clean unused data from reports. This is an optional step
+
+    ```bash
+        hab pkg exec chef/knife-ec-backup knife tidy server clean --backup-path /path/to/an-ec-backup
+    ```
+
+5. Execute the below command to copy the backup directory to the Automate HA Chef Server.
+
+    ```cmd
+        scp -i /path/to/key backup\_$(date '+%Y%m%d%H%M%s') user@host:/home/user
+    ```
+
+    If your HA Chef Server is in a private subnet, scp backup file to bastion and then to Chef Server.
 
 ## Restore Backed Up Data to Chef Automate HA
 
--   Execute the below command to install the habitat package for `knife-ec-backup`
+- Execute the below command to install the habitat package for `knife-ec-backup`
 
-```cmd
-    hab pkg install chef/knife-ec-backup
-```
+    ```cmd
+        hab pkg install chef/knife-ec-backup
+    ```
 
--   Execute the below command to restore the backup.
+- Execute the below command to restore the backup.
 
-```cmd
-    hab pkg exec chef/knife-ec-backup knife ec restore /home/centos/backup\_2021061013191623331154 -yes --concurrency 1 --webui-key /hab/svc/automate-cs-oc-erchef/data/webui\_priv.pem --purge -c /hab/pkgs/chef/chef-server-ctl/*/*/omnibus-ctl/spec/fixtures/pivotal.rb
-```
+    ```cmd
+        hab pkg exec chef/knife-ec-backup knife ec restore <path/to/directory/backup_file> -yes --concurrency 1 --webui-key /hab/svc/automate-cs-oc-erchef/data/webui\_priv.pem --purge -c /hab/pkgs/chef/chef-server-ctl/*/*/omnibus-ctl/spec/fixtures/pivotal.rb
+    ```
 
-## In place Migration (Chef Backend to Automate HA)
+## Steps to validate if Migration is successful
 
-As part of this scenario, customer will migrate from chef-backend (5 machines) to Automate HA in-place, i.e. Automate HA will be deployed in those 5 machines only where Chef-backend is running. One extra bastion node will be required which will be managing the deployment of Automate HA on the chef backend infrastructure.
+- Execute the below command from the old server where a knife is installed and from the new server where the knife is installed :
 
-{{< note >}}
+    ```cmd
+        curl https://raw.githubusercontent.com/chef/automate/main/dev/infra_server_objects_count_collector.sh -o infra_server_objects_count_collector.sh
+        bash infra_server_objects_count_collector.sh -S test -K Key -F Filename
+    ```
 
-This will require downtime, so plan accordingly. A reduced performance should be expected with this. 
+- `-S` is the Chef Server URL
+- `-K` is the path of pivotal.pem file
+- `-F` is the path to store the output file
+- Now run the below command to check the differences between the old and new data. Ideally, there should be no differences if the migration was done successfully.
 
-{{< /note >}}
+    ```cmd
+        diff old_server_file new_server_file
+    ```
+
+## In-place Migration (Chef Backend to Automate HA)
```

Comment:
We should add a section to migrate Chef Server/Backend with S3 to Automate HA. We can talk about two sub scenarios here:
- If we want to use the same S3 bucket
- If we want to use a different S3 bucket
The section should talk about steps to configure and migrate as well the implications of these two approaches

Reply:
We have added a section. Can you please have a look. @kalroy